volume-providers/vpc/vpcclient/models/constants.go

```diff
@@ -16,4 +16,10 @@ const (
 
 	// UserAgent identifies IKS to the RIaaS API
 	UserAgent = "IBM-Kubernetes-Service"
+
+	// GTypeClassic ...
+	GTypeClassic = "gc"
```

Comment:
Can be have a meaningful name like VPCClassic

Reply:
GenerationClassic is being used everywhere (riaas/riaas-provider/0. Let it be "gc" only NOT "vc"

Reply:
I am talking about variable name not the value as all doc is talking about VPCClassic not the generation classic

Reply:
its fine as we discussed yesterday